internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -355,14 +326,24 @@ func convertVolumes(input map[string]manifest.Volume) ([]*template.Volume, error
 		//   a) an EFS configuration, which must be valid
 		//   b) no EFS configuration, in which case the volume is created using task scratch storage in order to share
 		//      data between containers.
-		if volume.EFS != nil && volume.EFS.UseManagedFS() {
+
+		// If EFS is not configured, just add the name to create an empty volume and continue.
+		if volume.EFS == nil || volume.EFS.EmptyVolume() {
```

Comment:
Does this mean if someone specifies:
```yaml
volumes:
  myFS:
    efs:
```
We will create a scratch volume instead of an efs fs?

Reply:
ooh, yes. good catch.